Content.IntegrationTests/Tests/Utility/EntityWhitelistTest.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Content.Server.GameObjects.Components;
+using Content.Server.GameObjects.Components.Items.Storage;
+using Content.Shared.GameObjects.Components.Tag;
+using Content.Shared.Prototypes;
+using Content.Shared.Utility;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager;
+
+namespace Content.IntegrationTests.Tests.Utility
+{
+    [TestFixture]
+    [TestOf(typeof(EntityWhitelist))]
+    public class EntityWhitelistTest : ContentIntegrationTest
```

Comment:
this ideally wouldn't be an IntegrationTest but i need to be able to spawn entities properly and I couldn't see a way to do that in normal tests